.github/workflows/validation.yaml

```
@@ -94,3 +99,31 @@ jobs:
       - name: Run lint script
         run: npm run lint
         shell: bash
+
+  run-tests:
```

```suggestion
  # -------------------------------------------------------------------------------------------------------------------
  # RUN PRE-COMMIT VALIDATION
  # -------------------------------------------------------------------------------------------------------------------
  run-tests:
```

Would add a comment header here just to keep it consistent.